tests/pebble.rs

```diff
@@ -264,6 +264,35 @@ async fn order_deactivate() -> Result<(), Box<dyn StdError>> {
     Ok(())
 }
 
+/// Test account deactivation
+#[tokio::test]
+#[ignore]
+async fn acct_deactivate() -> Result<(), Box<dyn StdError>> {
```

Comment:
Nit: don't feel like abbreviating `account` in the test name is worth it.

Reply:
Fixed.